models/schemas/EstimateFee.yml

```diff
@@ -0,0 +1,11 @@
+type: object
```

Comment:
We don't need this model here if we're removing the fee estimate endpoint

Reply:
done